js/w3c/fixup.js

```diff
@@ -0,0 +1,40 @@
+
+// Module w3c/fixup
+// Inserts a link to the appropriate JS Extension for the W3C Spec Style Sheet.
+// CONFIGURATION
+//  - specStatus: the short code for the specification's maturity level or type (required)
+
+define(
+    ["core/utils"],
+    function (utils) {
+        return {
+            run:    function (conf, doc, cb, msg) {
+                msg.pub("start", "w3c/fixup");
+                if (!conf.specStatus) msg.pub("error", "Configuration 'specStatus' is not set, required for w3c/fixup");
+                var statStyle = conf.specStatus;
+                if (statStyle === "FPWD"    ||
```

Comment:
ah, that's the next big `if()`... this all seems sub-optimal. This should be handled by the `js/w3c/style.js` module.